fiftyone/server/mutation.py

```diff
@@ -292,15 +294,25 @@ async def create_saved_view(
         # view arg required to be an instance of
         # `fiftyone.core.view.DatasetView`
         if use_state:
+            print("stateful create_saved_view dataset:", dataset)
+            result_view = get_dataset_view(
```

Comment:
@kaixi-wang where is get_dataset_view defined? 🤔

Reply:
This needs to be renamed to `fiftyone.server.view.get_view()`. It was renamed in an upstream branch.

Reply:
brian renamed it to get_view in this pr that merged (https://github.com/voxel51/fiftyone/pull/2464/files)